sources/engine/Stride.Games/GameTime.cs

```diff
@@ -94,10 +97,44 @@ public GameTime(TimeSpan totalTime, TimeSpan elapsedTime)
         /// <value><c>true</c> if the <see cref="FramePerSecond"/> and <see cref="TimePerFrame"/> were updated for this frame; otherwise, <c>false</c>.</value>
         public bool FramePerSecondUpdated { get; private set; }
 
+
+
+        /// <summary>
+        /// Gets the amount of time elapsed multiplied by the time factor.
+        /// </summary>
+        /// <value>The warped elapsed time</value>
+        public TimeSpan WarpElapsed 
+        {
+            // TODO: When switching with .NET 5, use the multiply operator instead of this. elapsedGameTime * Factor
+            get => TimeSpan.FromSeconds(Elapsed.TotalSeconds * Factor);
```

Comment:
Use ``Elapsed.Ticks`` and ``TimeSpan.FromTicks`` instead, FromSeconds is less precise, see remarks over at https://docs.microsoft.com/en-us/dotnet/api/system.timespan.fromseconds?view=netframework-4.8